pandas/core/groupby/generic.py

```diff
@@ -1207,7 +1208,10 @@ def count(self):
 
         mask = (ids != -1) & ~isna(val)
         ids = ensure_platform_int(ids)
-        out = np.bincount(ids[mask], minlength=ngroups or 0)
+        minlength = ngroups or 0
```

Comment:
can u simplify this

Reply:
I've made this a one-liner in the newest update.